service/src/main/java/uk/gov/mca/beacons/api/DataSeeder.java

```diff
@@ -99,12 +99,14 @@ AccountHolder seedAccountHolder() {
   Beacon seedBeacon(AccountHolderId accountHolderId) {
     Beacon beacon = new Beacon();
     beacon.setBeaconStatus(BeaconStatus.NEW);
-    beacon.setAccountHolderId(accountHolderId);
+    beacon.setBeaconType("LAND");
```

Comment:
How come they're all land type?

Reply:
Just realised that the version on main doesn't set the type so will remove